src/test/app/DepositAuth_test.cpp

```diff
@@ -724,14 +797,553 @@ struct DepositPreauth_test : public beast::unit_test::suite
         }
     }
 
+    void
+    testCredentialsPayment()
+    {
```

Comment:
There should be an additional test here for an account that has been DepositAuth-ed the account way but still provides credentials.

Reply:
Don't think we need this as it is the same as wrong credentials / depositPreauth doesn't exist

Reply:
I'd like to change that behavior - it feels unintuitive and unexpected. If credentials are ignored if DepositAuth doesn't exist, they should also be ignored if DepositAuth exists and is unnecessary.